### ISSUE TITLE
Annotating Files

### ISSUE THREAD

**Issue by ArthurMousatov** [filebrowser/filebrowser#2664]
**Is your feature request related to a problem? Please describe.**

I want to be able to annotate files. E.G. adding descriptions to images and such.

**Describe the solution you'd like**

A description textbox that allows me to add extra information about a file. It would be nice to have the description shown alongside images/videos.

**Describe alternatives you've considered**

Adding a text file listing file -> description
Adding titles to files.
**Additional context**

Here is what google drive has: 

![image](https://github.com/filebrowser/filebrowser/assets/57199800/db8a45a6-702f-4a10-a9ff-ce5c7a9f5479)

I'm down to implement this, but I would want input (and maybe I missed a PR/existing feature already in place) before starting.
